js/src/elements/form-behavior.ts

```diff
@@ -375,6 +384,11 @@ export class FormBehaviorElement extends HTMLElement {
 				continue;
 			}
 
+			// Don't check markdown editor attachment input
```

Comment:
No need to hold this, but consider adding an attribute instead of a class. `data-skip-validation` or something similar.

Reply:
Updated to check its id for now. I'll have to find the markdown editor codebase to add data attributes later.